server/service/apple_mdm.go

```diff
@@ -551,6 +551,60 @@ func (svc *Service) GetMDMAppleProfilesSummary(ctx context.Context, teamID *uint
 	return ps, nil
 }
 
+type getMDMAppleFileVaultSummaryRequest struct {
+	TeamID *uint `query:"team_id,optional"`
+}
+
+type getMDMAppleFileVauleSummaryResponse struct {
+	Applied             uint  `json:"applied"`
+	ActionRequired      uint  `json:"action_required"`
+	Enforcing           uint  `json:"enforcing"`
+	Failed              uint  `json:"failed"`
+	RemovingEnforcement uint  `json:"removing_enforcement"`
+	Err                 error `json:"error,omitempty"`
+}
+
+// QUESTION: what is this line for?
+func (r getMDMAppleFileVauleSummaryResponse) error() error { return r.Err }
+
+// QUESTION: `errorer` naming? why this name?
+func getMdmAppleFileVaultSummaryEndpoint(ctx context.Context, request interface{}, svc fleet.Service) (errorer, error) {
+	req := request.(*getMDMAppleFileVaultSummaryRequest)
+
+	fvs, err := svc.GetMDMAppleFileVaultSummary(ctx, req.TeamID)
+	if err != nil {
+		return &getMDMAppleFileVauleSummaryResponse{Err: err}, nil
+	}
+
+	return &getMDMAppleFileVauleSummaryResponse{
+		Applied:             fvs.Applied,
+		ActionRequired:      fvs.ActionRequired,
+		Enforcing:           fvs.Enforcing,
+		Failed:              fvs.Failed,
+		RemovingEnforcement: fvs.RemovingEnforcement,
+	}, nil
+}
+
+// QUESTION: workflow for developing new APIs? whats your setup quickly test code working?
+func (svc *Service) GetMDMAppleFileVaultSummary(ctx context.Context, teamID *uint) (*fleet.MDMAppleFileVaultSummary, error) {
+	if err := svc.authz.Authorize(ctx, fleet.MDMAppleConfigProfile{TeamID: teamID}, fleet.ActionRead); err != nil {
+		return nil, ctxerr.Wrap(ctx, err)
+	}
+
+	fvs, err := svc.ds.GetMDMAppleFileVaultSummary(ctx, teamID)
+	if err != nil {
+		return nil, ctxerr.Wrap(ctx, err)
+	}
+
+	return &fleet.MDMAppleFileVaultSummary{
```

Comment:
No need to copy the values in another `fleet.MDMAppleFileVaultSummary` struct, `fvs` is already of that type, you can just `return fvs, nil`.